basic-features.md

```diff
@@ -1,80 +1,136 @@
+| `utf8mb4_general_ci`                                                                                     | Experimental | Experimental | Experimental | N            | N            | N            |
+| `utf8mb4_unicode_ci`                                                                                     | Experimental | Experimental | Experimental | N            | N            | N            |
+| `ascii_bin`                                                                                              | Y            | Y            | Y            | Y            | Y            | Y            |
+| `latin1_bin`                                                                                             | Y            | Y            | Y            | Y            | Y            | Y            |
+| `binary`                                                                                                 | Y            | Y            | Y            | Y            | Y            | Y            |
+| **Indexing and constraints**                                                                             | **5.1**      | **5.0**      | **4.0**      | **3.1**      | **3.0**      | **2.1**      |
+| [Expression indexes](/sql-statements/sql-statement-create-index.md#expression-index)                     | Experimental | Experimental | Experimental | N            | N            | N            |
+| [Columnar storage (TiFlash)](/tiflash/tiflash-overview.md)                                               | Y            | Y            | Y            | Y            | N            | N            |
+| [RocksDB engine](/storage-engine/rocksdb-overview.md)                                                    | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Titan plugin](/storage-engine/titan-overview.md)                                                        | Y            | Y            | Y            | Experimental | Experimental | Experimental |
+| [Invisible indexes](/sql-statements/sql-statement-add-index.md)                                          | Y            | Y            | N            | N            | N            | N            |
+| [Composite `PRIMARY KEY`](/constraints.md)                                                               | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Unique indexes](/constraints.md)                                                                        | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Clustered index on integer `PRIMARY KEY`](/constraints.md)                                              | Y            | Y            | Y            | Y            | Y            | Y            |
+| [Clustered index on composite or non-integer key](/constraints.md)                                       | Experimental | Experimental | N            | N            | N            | N            |
```

Comment:
@xiongjiwei Can you please review this section (Indexing and contraints)?

Reply:
I think `alter primary key` is deprecated from 5.0.

Reply:
`Clustered index` is GA on v5.0 and v5.1, so please change them to 'Y'.